docs/proto_template.tmpl

```diff
@@ -2,23 +2,57 @@
 sidebar_position: 60
 toc_max_heading_level: 4
```

Comment:
Since the heading `# Protocol documentation` was moved to further down in the file, the sidebar title was defaulting to `proto`. This adds back the correct sidebar title.

```suggestion
title: Protocol documentation
toc_max_heading_level: 4
```

Reply:
Ah, good catch!